ui/src/serverIO.js

```diff
@@ -413,8 +413,7 @@ class ServerIO {
     });
 
     this.hwrSocket.on('forceSignout', () => {
-      this.dispatch(forcedSignout());
-      serverIO.disconnect();
```

Comment:
Don't think we want to disconnect the WS connection, since we connect _before_ logging in, in `App`.